src/components/App/App.tsx

```diff
@@ -35,7 +35,7 @@ function App() {
 
   const [user, setUser] = useState<any>(activeUser); //Holds the current user
   const [savedArticles, setSavedArticles] = useState<string[]>(savedArts);
-  const [pets, setPets] = useState<any[]>(PetsStorage);
```

Comment:
I resolved the localStorage issues -- we can migrate back to that to lessen the cost of the fetch's